UI-UX Design/README.md

```diff
@@ -283,6 +283,18 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
       <td><a href="https://www.youtube.com/watch?v=2OzTi0iUMlE">UI/UX Design Job Interview Questions and Answers</a></td>
       <td>Questions that you will most likely be asked in a UX design interview as well as going over tips on how best to answer them.</td>
       </tr>
+      <tr>
+      <td><a href="https://sensible.com/dont-make-me-think/">"Don't Make Me Think" by Steve Krug (Book)</a></td>
+      <td> "Don't Make Me Think" is a classic book on web usability and user experience design. It provides valuable insights into creating user-friendly interfaces and emphasizes the importance of designing for intuitive user interactions. Steve Krug's writing is both practical and easy to understand, making it an excellent resource for UX designers.</td>
+      </tr>
```

Comment:
This is more appropriate in the [book section](https://github.com/jfmartinz/ResourceHub/tree/main/UI-UX%20Design#books). You can remove this or just add it to the book section. and if you're going to add it on the book section, don't forget to add a status if it's paid or free